src/Http/Controllers/VoyagerController.php

```diff
@@ -101,4 +107,10 @@ public function assets(Request $request)
 
         return response('', 404);
     }
+
+    protected function userCannotUploadImageIn($dataType, $action)
+    {
+        return auth()->user()->cannot($action, app($dataType->model_name))
```

Comment:
Since we're in a controller class, any reason not to use `$this->authorize(...)` instead?

https://laravel.com/docs/8.x/authorization#via-controller-helpers

Reply:
If I'm not mistaken authorize does not return a Boolean so we won't be able to check both cases.
I'll verify tomorrow

Reply:
Yes, authorize throws an AuthorizationException so we can't use it in this case.